.github/workflows/net-perf-gke.yaml

```diff
@@ -103,6 +103,30 @@ jobs:
             encryption: "ipsec"
             hubble: "true"
 
+          - index: 9
```

Comment:
Nit. I wonder if it could make sense to re-number the tests so that all the ones without hubble come first, and then the others follow, for consistency. AFAICT, the index is not used when exporting the data, hence it should not create problems from that point of view.